include/call.hpp

```diff
@@ -291,7 +291,8 @@ class call : virtual public task, virtual public listener, public virtual socket
 
     char * get_header_field_code(const char * msg, const char * code);
     char * get_last_header(const char * name);
-    char * get_last_request_uri();
+    char * get_last_tag (const char *name);
+    char * get_last_request_uri(const char *name);
```

Comment:
This should not be called `get_last_request_uri` but `get_last_uri_from` or something.